teps/0044-more-execution-options-for-pipelines.md

```diff
@@ -157,6 +155,7 @@ See [TEP-0074](./0074-deprecate-pipelineresources.md) for the deprecation plan f
     - This requirement is being included because we could choose a solution that doesn't
       address the above use case; for example in PipelineResources, you can have a
       storage "output" but if the steps fail, the "output" pipelineresource will not run
+- Execution options must be configurable at runtime, but we may choose to make them configurable at authoring time as well.
```

Comment:
to clarify, will this tep provide the execution options at runtime only? just want to confirm that the runtime configuration only is in scope for this tep

Reply:
updated after wordsmithing offline 👍